source/utilities.tex

```diff
@@ -15967,6 +15972,399 @@
+
+\pnum
+\throws
+Any exception thrown by the initialization of the target object.
+May throw \tcode{bad_alloc} unless \tcode{VT} is
+a function pointer or a specialization of \tcode{reference_wrapper}.
+\end{itemdescr}
+
+\indexlibrarymember{operator=}{move_only_function}%
+\begin{itemdecl}
+move_only_function& operator=(move_only_function&& f);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to: \tcode{move_only_function(std::move(f)).swap(*this);}
+
+\pnum
+\returns
+\tcode{*this}.
+\end{itemdescr}
+
+\indexlibrarymember{operator=}{move_only_function}%
+\begin{itemdecl}
+move_only_function& operator=(nullptr_t) noexcept;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Destroys the target object of \tcode{*this}, if any.
+
+\returns
+\tcode{*this}.
+\end{itemdescr}
+
+\indexlibrarymember{operator=}{move_only_function}%
+\begin{itemdecl}
+template<class F> move_only_function& operator=(F&& f);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to: \tcode{move_only_function(std::forward<F>(f)).swap(*this);}
+
+\pnum
+\returns
+\tcode{*this}.
+\end{itemdescr}
+
+\indexlibrarydtor{move_only_function}%
+\begin{itemdecl}
+~move_only_function();
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Destroys the target object of \tcode{*this}, if any.
+\end{itemdescr}
+
+\rSec4[func.wrap.move.inv]{Invocation}
+
+\indexlibrarymember{operator bool}{move_only_function}%
+\begin{itemdecl}
+explicit operator bool() const noexcept;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\returns
+\tcode{true} if \tcode{*this} has a target object, otherwise \tcode{false}.
+\end{itemdescr}
+
+\indexlibrarymember{operator bool}{move_only_function}%
+\begin{itemdecl}
+R operator()(ArgTypes... args) @\cv{}@ @\placeholder{ref}@ noexcept(@\placeholder{noex}@);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\expects
+\tcode{*this} has a target object.
+
+\pnum
+\effects
+Equivalent to:
+\begin{codeblock}
+return @\placeholder{INVOKE}@<R>(static_cast<F @\placeholder{inv-quals}@>(f), std::forward<ArgTypes>(args)...);
+\end{codeblock}
+where \tcode{f} is the target object of \tcode{*this} and
+\tcode{f} is lvalue of type \tcode{F}.
+\end{itemdescr}
+
+\rSec4[func.wrap.move.util]{Utility}
+
+\indexlibrarymember{swap}{move_only_function}%
+\begin{itemdecl}
+void swap(move_only_function& other) noexcept;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Exchanges the targets of \tcode{*this} and \tcode{other}.
+\end{itemdescr}
+
+\indexlibrarymember{swap}{move_only_function}%
+\begin{itemdecl}
+friend void swap(move_only_function& f1, move_only_function& f2) noexcept;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to: \tcode{f1.swap(f2)}.
+\end{itemdescr}
+
+\indexlibrarymember{operator==}{move_only_function}%
+\begin{itemdecl}
+friend bool operator==(const move_only_function& f, nullptr_t) noexcept;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\returns
+\tcode{true} if \tcode{f} has no target object.
```

Comment:
```suggestion
\tcode{true} if \tcode{f} has no target object, otherwise \tcode{false}.
```

Reply:
This is not in the incoming paper.

http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2021/p0288r9.html#Specification

Reply:
Fixed.